src/store.js

```diff
@@ -0,0 +1,15 @@
+import { createStore } from 'redux'
+import { todoApp } from './reducers'
+
+const initialState = {
+  squareColor: 'black',
```

Comment:
And actually black is the colour, it is better to add some text like 'Colour isn't picked' inside the box